src/sentry/integrations/utils/code_mapping.py

```diff
@@ -15,6 +15,13 @@
 # We only care about the ones that would show up in stacktraces after symbolication
 EXTENSIONS = ["js", "jsx", "tsx", "ts", "mjs", "py", "rb", "php", "go"]
 
+# List of file paths prefixes that should become stack trace roots
+FILE_PATH_PREFIX_LENGTH = {
+    "app:///": 7,
+    "../": 3,
+    "./": 2,
+}
```

Comment:
Dictionary of file path prefixes and their lengths (to make removing them later easier)